lib/resty/timerng/thread/worker.lua

```diff
@@ -165,6 +168,8 @@ local function thread_body(context, self,
             goto continue
         end
 
+        utils_table_clear(ngx.ctx)
```

Comment:
Will this affect the current request? It seems like a resty library is manipulating the current request ctx.

Reply:
This line always runs in the `timer` context, I think the request context should not be accessed in a timer.

Reply:
clear, the timer has its own request ctx.